doc/interop-test-descriptions.md

```diff
@@ -452,10 +466,9 @@ Asserts:
 
 ### unimplemented_method
 
-Status: Not yet implementable
+Status: Ready for implementation. Blocking beta.
```

Comment:
It is not implementable. The client is expected to do things it can't do (UnimplementedCall accepts an Empty).

We should probably also specify the service is UnimplementedService.